compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/BallerinaParserErrorHandler.java

```diff
@@ -3441,6 +3443,17 @@ protected ParserRuleContext getNextRule(ParserRuleContext currentCtx, int nextLo
                 return getNextRuleForBindingPattern();
             case TUPLE_MEMBERS:
                 return ParserRuleContext.TUPLE_MEMBER;
+            case ON_FAIL_CHECK:
+                return ParserRuleContext.ON_KEYWORD;
+            case ON_KEYWORD:
```

Comment:
We already have an entry for next rule for `ON_KEYWORD` here
https://github.com/ballerina-platform/ballerina-lang/blob/217853c58bfb1aef5b069c716a2dfcff5c88d129/compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/BallerinaParserErrorHandler.java#L3857-L3867